cmd/templ/lspcmd/lsp_test.go

```diff
@@ -100,12 +99,15 @@ func TestCompletion(t *testing.T) {
 		assert      func(t *testing.T, cl *protocol.CompletionList) (msg string, ok bool)
 	}{
 		{
-			line:        13,
-			replacement: ` <div data-testid="count">{  `,
-			cursor:      `                            ^`,
+			line:        3,
+			replacement: `templ`,
+			cursor:      `    ^`,
 			assert: func(t *testing.T, actual *protocol.CompletionList) (msg string, ok bool) {
-				if diff := lspdiff.CompletionList(nil, actual); diff != "" {
-					return fmt.Sprintf("unexpected completion: %v", diff), false
+				if !lspdiff.CompletionListContainsText(actual, "templ") {
+					return fmt.Sprintf(
```

Comment:
Would really prefer that this formatting wasn't applied.

Reply:
I'll figure out a whitelist, I keep forgetting to toggle it.

Reply:
Ah, sorted, it wasn't gofumpt after all, I had the golines formatter enabled too.